DigitalLearningSolutions.Web/ViewComponents/NumberOfAdministratorsViewComponent.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace DigitalLearningSolutions.Web.ViewComponents
+{
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Web.ViewModels.Common;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class NumberOfAdministratorsViewComponent : ViewComponent
+    {
+        private readonly ICentresDataService centresDataService;
+        private readonly IUserDataService userDataService;
+
+        public NumberOfAdministratorsViewComponent(
+            ICentresDataService centresDataService,
+            IUserDataService userDataService
+        )
+        {
+            this.centresDataService = centresDataService;
+            this.userDataService = userDataService;
+        }
+
+        /// <summary>
+        /// Render NumberOfAdministrators view component.
+        /// </summary>
+        /// <param name="centreId"></param>
+        /// <returns></returns>

```

Comment:
Not sure the comment is necessary in this case, as I think the code is really clear and straight forward 👍